developer.bat

```diff
@@ -0,0 +1,17 @@
+IF NOT EXIST build mkdir build
+
+cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=install -G "Visual Studio 16 2019" -DDEVELOPER=TRUE ..
+
+cmake --build . --config Debug --target install
```

Comment:
I think there is no need to perform the building and installation itself, it is enough to just configure the project to generate .sln

Reply:
The proposed approach in this PR is to add an audacity-target post-build step to copy the executable to the install directory. If you just build audacity without building install first, it will copy it, it will be alone in its `install/bin`.